src/NotepadNext/dialogs/MainWindow.cpp

```diff
@@ -521,21 +521,22 @@ MainWindow::MainWindow(NotepadNextApplication *app) :
                                 .arg(APP_VERSION, QStringLiteral(APP_COPYRIGHT).toHtmlEscaped()));
     });
 
-    LuaConsoleDock *luaConsoleDock = new LuaConsoleDock(app->getLuaState(), this);
-    luaConsoleDock->hide();
-    addDockWidget(Qt::BottomDockWidgetArea, luaConsoleDock);
-    ui->menuHelp->addSeparator();
-    ui->menuHelp->addAction(luaConsoleDock->toggleViewAction());
+    QAction *separator = ui->menuHelp->insertSeparator(ui->actionCheckForUpdates);
+
+    EditorInspectorDock *editorInspectorDock = new EditorInspectorDock(this);
+    editorInspectorDock->hide();
+    addDockWidget(Qt::RightDockWidgetArea, editorInspectorDock);
+    ui->menuHelp->insertAction(separator, editorInspectorDock->toggleViewAction());
 
     LanguageInspectorDock *languageInspectorDock = new LanguageInspectorDock(this);
     languageInspectorDock->hide();
     addDockWidget(Qt::RightDockWidgetArea, languageInspectorDock);
-    ui->menuHelp->addAction(languageInspectorDock->toggleViewAction());
+    ui->menuHelp->insertAction(editorInspectorDock->toggleViewAction(), languageInspectorDock->toggleViewAction());
 
-    EditorInspectorDock *editorInspectorDock = new EditorInspectorDock(this);
-    editorInspectorDock->hide();
-    addDockWidget(Qt::RightDockWidgetArea, editorInspectorDock);
-    ui->menuHelp->addAction(editorInspectorDock->toggleViewAction());
+    LuaConsoleDock *luaConsoleDock = new LuaConsoleDock(app->getLuaState(), this);
+    luaConsoleDock->hide();
+    addDockWidget(Qt::BottomDockWidgetArea, luaConsoleDock);
+    ui->menuHelp->insertAction(languageInspectorDock->toggleViewAction(), luaConsoleDock->toggleViewAction());
```

Comment:
Just a thought, would it be cleaner to use a single call to [insertActions()](https://doc.qt.io/qt-5/qwidget.html#insertActions) something like

```cpp
ui->menuHelp->insertActions(separator, { ___, ___, ___});
```

Haven't used that myself so not sure if there's some technical reason why it isn't viable.

Reply:
Do you mean constructing the action object while passing it to insertAction?
InsertActions takes a QAction as a second parameter, but we construct a LuaConsoleDock, which is a QDockWidget. After that hide() needs to be called on it, then we pass to insertAction using the toggleViewAction().
I don't think it could make it simpler or even possible.

Reply:
That's fine. I'll use the code as is. I'd rather have something working and clean it up later.